### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by d-makarenko**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @SergeyTeslov

### CODE REVIEW

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        }
+        static void Main(string[] args)
+        {
+
+            //while (true) 
+            //{
+            //    functionNumberOfDays(); 
+            //}
+            functionNumberOfDays();
+
```
Comment (d-makarenko) [inline ConsoleApp1/ConsoleApp1/Program.cs:56]: Мы еще не изучали функции, так не пойдет

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+using System;
+public enum daysOfTheWeek
+{
+    Monday    = 1,
+    Tuesday   = 2,
+    Wednesday = 3,
+    Thursday  = 4,
+    Friday    = 5,
+    Saturday  = 6,
+    Sunday    = 7
```
Comment (d-makarenko) [inline ConsoleApp1/ConsoleApp1/Program.cs:12]: Мы еще не изучали enum

**Comment by d-makarenko** [comment]
Необходимо внести исправления по замечаниям.

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            var userInput = Console.ReadLine();
+
+            //проверка введёных данных 
+            //start            
+            Int32 val;
+            if (!Int32.TryParse(userInput, out val))
+            {
+                Console.WriteLine("not a number entered");
+                return;
+            }
```
Comment (d-makarenko) [inline ConsoleApp1/ConsoleApp1/Program.cs:32]: Предлагаю заменить это на обработчик исключений

```suggestion
            try {
                val = int.Parse(userInput);
            }
            except {
                Console.WriteLine("not a number entered");
                return;
            }
```

Comment (SergeyTeslov) [reply ConsoleApp1/ConsoleApp1/Program.cs:32]: ой а что такое обработчик исключений?

Comment (d-makarenko) [reply ConsoleApp1/ConsoleApp1/Program.cs]: Это из будущих лекций, просто закоммить изменения

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+            var userInput = Console.ReadLine();
+
+            //проверка введёных данных 
+            //start            
+            Int32 val;
+            try {
+                val = int.Parse(userInput);
+            }
+            except {
```
Comment (d-makarenko) [inline ConsoleApp1/ConsoleApp1/Program.cs:31]: это ключевое слово из python, нужно заменить на catch
```c#
      try {
      ...
      }
      catch {
      }
```

Comment (SergeyTeslov) [reply ConsoleApp1/ConsoleApp1/Program.cs:31]: а уведомления мне так и не приходят в github

**Comment by d-makarenko** [comment]
@SergeyTeslov прошу исправить замечания!

**Comment by d-makarenko** [comment]
Нужно исправить ошибку

**Comment by d-makarenko** [comment]
Программа все еще не компилируется

**Comment by d-makarenko** [comment]
@SergeyTeslov исправляй!

**Pull request merged**